davidshare/Fast-Food-Fast
# #160498831 Remove code climate test-coverage from repository

#### What does this PR do?
Remove codeclimate test coverage from repository

#### Description of Task to be completed?
 - remove code climate token from travis.yml file
 - change the format for the test script
 - remove markup for code climate test-coverage badge from repository

#### How should this be manually tested?
- visit the repository on GitHub and check the readme file for the test coverage badge

#### What are the relevant pivotal tracker stories?
[#160498831](https://www.pivotaltracker.com/story/show/160498831)
